lib/active_model_serializers/model.rb

```diff
@@ -0,0 +1,43 @@
+# ActiveModelSerializers::Model is a convenient
```

Comment:
So, it is essentially executable documentation.  It's mostly the old 'Model' fixture prettied up a bit.  And this wasn't the first time we've had to answer this question.  Now the answer is tested regularly, too, like how some testing models or files are in lib in rails and other gems.  E.g activejob::test

B mobile phone

> On Oct 9, 2015, at 8:26 AM, Lucas Hosseini <notifications@github.com> wrote:
> 
> In lib/active_model_serializers/model.rb:
> 
> > @@ -0,0 +1,43 @@
> > +# ActiveModelSerializers::Model is a convenient
> I see the value in this, but maybe it belongs in the docs, rather than in the code?
> 
> —
> Reply to this email directly or view it on GitHub.
>